SU-SWS/linky_clicky
# Display Suite

# READY FOR REVIEW

# Summary
- Beefs up the Scenario for `ds` to check the "View mode per node" functionality

# Needed By (Date)
- Whenever

# Criticality
- How critical is this PR on a 1-10 scale? 1

# Steps to Test

1. Check out this branch
2. Run this Scenario against a site with the `Stanford` profile
3. Ensure that it passes

# Affected Projects or Products
- Tangentially, everything. But very few sites use this functionality.

# Associated Issues and/or People
- Any other contextual information that might be helpful: this came about because of the [DS 7.x-2.15 security update](https://www.drupal.org/sa-contrib-2018-019)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
